### PR TITLE
Closes #6

### LINKED ISSUE #6: Clean up styles and fonts
From @jasonscharff:

1. Not a huge fan of the font. I think the logo goes for cleaner sort of look, but the font and the dollar sign in front of the sections looks more "code like." I prefer the cleaner look. The font in the logo is Ubuntu from google fonts. Love the new white logo with red sun BTW.
2. I'd like to add some sort of navigation bar as it could get difficult to follow with real content
3. Add a sponsor section––For now I'll just add something like for info on sponsoring email sponsor@totalityhacks.com
4. The copyright should actually be "Hackathon Organizers, LLC"
5. I think we should shrink the footer––and not have links to things that don't exist.
6. I'd like more distinction between the sections through different backgrounds or something. It can get hard to read in long form.

### PR DESCRIPTION
**Pull request by gmittal**
Closes #6. Still waiting for content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 <img id="profpic" src="logo.svg" />
                 <center>
                   <p class="lead">
-		    The world's largest hackathon. Motto.<br /><br />
+		    Redefning the hackathon.<br /><br />
```
Comment (gmittal) [inline index.html]: Spelling

File: index.html [hunk 2]
```diff
@@ -97,11 +92,32 @@ <h3>Workshops</h3>
...
+		      <p>
+                    We don't want financial cost to be a barrier to attending Totality Hacks. If you're skilled, we want you. As such, we'll do our best in providing travel reimbursement. Along with your decision, you'll also receive a travel reimbursement cap.
+              </p>
+            
+             <br />
 
 		
 		<h1 id="sponsors">Sponsors</h1>
-		<p>Contact <a href="mailto:sponsor@totalityhacks.com">sponsor@totalityhacks.com</a>.</p>
+		<p>Sponsoring TotalityHacks allows us to put on an event that empowers hackers be hackers. If you or your company is interested in sponsoring TotalityHacks, please email <a href="mailto:sponsor@totalityhacks.com">sponsor@totalityhacks.com</a>.</p>
```
Comment (gmittal) [inline index.html]: 'hackers be hackers' to 'hackers to be hackers'

**Comment by jasonscharff** [comment]
@jreinstra Take a look.

**Comment by gmittal** [comment]
Added MLH Code of Conduct in de665a4

**Comment by jasonscharff** [comment]
@gmittal Make it open in a new tab and then go ahead and merge.

**Pull request merged**